frontend/src/compliance_reporting/ComplianceReportingEditContainer.js

```diff
@@ -614,18 +615,32 @@ class ComplianceReportingEditContainer extends Component {
           <FontAwesomeIcon icon="info-circle" />
         </Tooltip>
       </h3>,
-      <p className="schedule-organization-address" key="organization-address">
-        {organizationAddress &&
-        AddressBuilder({
+      <br/>,
+      <p className="schedule-organization-address" key="organization-address"> 
+        {organizationAddress
+      ? ["Head Office:", AddressBuilder({
           address_line_1: organizationAddress.addressLine1,
           address_line_2: organizationAddress.addressLine2,
           address_line_3: organizationAddress.addressLine3,
           city: organizationAddress.city,
           state: organizationAddress.state,
           postal_code: organizationAddress.postalCode
-        })
+        })]:null
         }
       </p>,
+      <p className="schedule-organization-address" key="organization-attorney-address">
+      {organizationAddress.attorneyAddressOther ? 
+      ["B.C. Attorney Office: ",
+      AddressBuilder({
+        address_line_1: organizationAddress.attorneyAddressOther,
+        address_line_2: organizationAddress.attorneyStreetAddress,
+        address_line_3: organizationAddress.addressLine3,
```

Comment:
does address_line_3 still exist in the release branch? We may have to clean this up.